apps/explorer/src/pages/transaction-result/transaction-summary/ObjectChanges.tsx

```diff
@@ -0,0 +1,322 @@
+                                    </Text>
+                                    {locationIdType ===
+                                        LocationIdType.AddressOwner && (
+                                        <AddressLink address={ownerAddress} />
+                                    )}
+                                    {locationIdType ===
+                                        LocationIdType.ObjectOwner && (
+                                        <ObjectLink objectId={ownerAddress} />
+                                    )}
+                                    {locationIdType ===
+                                        LocationIdType.Shared && (
+                                        <ObjectLink
+                                            objectId={ownerAddress}
+                                            label="Shared"
+                                        />
+                                    )}
+                                </div>
+                            )
+                        }
+                    >
+                        <TransactionBlockCardSection
+                            title={
+                                <Text
+                                    variant="body/semibold"
+                                    color="steel-darker"
+                                >
+                                    {title}
+                                </Text>
+                            }
+                        >
+                            <ExpandableList
+                                items={changes.map(
+                                    ({ objectId, objectType }) => (
+                                        <ObjectDetail
+                                            key={objectId}
+                                            objectId={objectId}
+                                            objectType={objectType}
+                                        />
+                                    )
+                                )}
+                                defaultItemsToShow={DEFAULT_ITEMS_TO_SHOW}
+                                itemsLabel="Objects"
+                            >
+                                <div className={clsx(expandableListClassName)}>
+                                    <ExpandableListItems />
+                                </div>
+                            </ExpandableList>
+                        </TransactionBlockCardSection>
+                    </TransactionBlockCard>
+                );
+            })}
+        </>
+    );
+}
+
+interface ObjectChangesProps {
+    objectSummary: {
+        mutated: SuiObjectChangeMutated[];
+        created: SuiObjectChangeCreated[];
+        transferred: SuiObjectChangeTransferred[];
+    };
+}
+
+export function ObjectChanges({ objectSummary }: ObjectChangesProps) {
+    if (!objectSummary) {
+        return null;
+    }
+
+    const createdChangesByOwner = getGroupByOwner(objectSummary?.created);
```

Comment:
This was an issue but the PR was already merged so I have to do this separately. I am assuming we will need to change this in the wallet too since I do not see the similar method there.